admin/install_server.sh

```diff
@@ -21,6 +21,11 @@ echo "client_encoding = utf8" >> "$PG_CONF"
 
 service postgresql restart
 
+# Less compiler
+curl -sL https://deb.nodesource.com/setup | sudo bash -
```

Comment:
I'm tempted to use node-less for now, since it works, and involves less magic. Let's re-address this if we need advanced support in the future